lib/reply.js

```diff
@@ -50,7 +49,7 @@ Reply.prototype.send = function (payload) {
     setImmediate(
       wrapReplyEnd,
       this,
-      safeStringify(payload.output.payload)
+      JSON.stringify(payload.output.payload)
```

Comment:
Here you can use just `stringify`, it is declared at line [10](https://github.com/fastify/fastify/blob/fix-178/lib/reply.js#L10).